### PR TITLE
Added Lucky Dip to LBP3 & changed category desc

### PR DESCRIPTION
**Pull request by W0lf4llo**
This is my first time developing something, this is awesome!! and i cant believe sumo didnt add Lucky Dip before D:

### CODE REVIEW

**Review by daggintosh — APPROVED** [review]
Looks good to me, and now I shall pass this on to the **Slendy** for approval. @Slendy

**Review by Slendy — APPROVED** [review]
LGTM, thanks for your contribution to Project Lighthouse!

File: ProjectLighthouse/Levels/Categories/LuckyDipCategory.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+using Microsoft.EntityFrameworkCore;
+
+namespace LBPUnion.ProjectLighthouse.Levels.Categories;
+
+public class LuckyDipCategory : Category
+{
+    public override string Name { get; set; } = "Lucky Dip";
+    public override string Description { get; set; } = "Randomized uploaded content";
+    public override string IconHash { get; set; } = "g820605";
+    public override string Endpoint { get; set; } = "luckydip";
```
Comment (Slendy) [inline ProjectLighthouse/Levels/Categories/LuckyDipCategory.cs:16]:
```suggestion
    public override string Endpoint { get; set; } = "lbp2luckydip";
```
This needs to match the endpoint name in SlotsController.cs which is `[HttpGet("slots/lbp2luckydip")]`
Either you need to add another `[HttpGet("slots/luckydip")]` or change this back to lbp2luckydip

**Review by Slendy — CHANGES_REQUESTED** [review]

File: ProjectLighthouse/Levels/Categories/LuckyDipCategory.cs (hunk 1 above)
Comment (W0lf4llo) [reply ProjectLighthouse/Levels/Categories/LuckyDipCategory.cs]: I fixed it :)

**Review by Slendy — APPROVED** [review]
LGTM

**Pull request merged**